src/main/java/rife/bld/operations/RunOperation.java

```diff
@@ -52,6 +54,22 @@ public RunOperation fromProject(BaseProject project) {
         if (project.usesRife2Agent()) {
             operation.javaOptions().javaAgent(project.getRife2AgentFile());
         }
+
+        // parse the run arguments if any
+        var args = project.arguments();
```

Comment:
Not sure about this, the intended use of the arguments is that each operation consumes the next arguments it needs, not any arguments anywhere passed to the command line. By only consuming the next ones, it's possible to freely chain commands on the command line without them interfering with each other

Reply:
I'm not sure what you mean. It only deals with `./bld/run` arguments as far as my testing went.  e.g.;

`./bld co run -args="blah" blah publish`

will fail with bld complaining about blah being an invalid argument.